packages/shared/styles/components/atoms/SfPrice.scss

```
@@ -4,25 +4,19 @@
   &__value {
     margin: var(--price-margin);
     color: var(--price-color, var(--c-text));
-    font: var(
+    text-decoration: var(--price-text-decoration);
+    @include font(
       --price-font,
-      font(
-        --price,
-        400,
-        var(--font-size-big),
-        1.6,
-        var(--body-font-family-primary)
-      )
+      var(--font-weight-bold),
+      var(--font-size-big),
+      1.6,
+      var(--body-font-family-primary)
```

```suggestion
      var(--body-font-family-secondary)
```
on the design, we use `Raleway` for price